Filters/FilterType.cs

```diff
@@ -23,6 +23,7 @@ public enum FilterType
         /// The command is a slider or input field
         /// </summary>
         RANGE = 32,
-        TEXT = 64
+        TEXT = 64,
+        EQUAL_NO_SEARCH = 128
```

Comment:
```suggestion
        EQUAL_NO_SEARCH = 129
```
Make it backward compatible so outdated UIs would still render it as drop-down

Reply:
What does changing 128 to 129 achieve? That breaks the use of flags doesnt it?
Also: Outdated UIs dont break, as the drop-down-selects are currently choosen if nothing else works

Reply:
129 is both (SIMPLE and EQUAL)
This use case is precisely why I implemented this with flags in the first place. 
that your ui isn't breaking is an implementation-detail of your UI it's not safe to assume that all other UIs/scripts using the filter system did it the same way.